keyboards/handwired/erikpeyronson/erkbd/erkbd.c

```diff
@@ -0,0 +1,68 @@
+// Copyright 2025 Erik Peyronson @erikpeyronson
+// SPDX-License-Identifier: GPL-3.0-or-later
+#include QMK_KEYBOARD_H
+
+static void oled_render_logo(void) {
+    // clang-format off
+    static const char PROGMEM qmk_logo[] = {
+        0x80, 0x81, 0x82, 0x83, 0x84, 0x85, 0x86, 0x87, 0x88, 0x89, 0x8A, 0x8B, 0x8C, 0x8D, 0x8E, 0x8F, 0x90, 0x91, 0x92, 0x93, 0x94,
+        0xA0, 0xA1, 0xA2, 0xA3, 0xA4, 0xA5, 0xA6, 0xA7, 0xA8, 0xA9, 0xAA, 0xAB, 0xAC, 0xAD, 0xAE, 0xAF, 0xB0, 0xB1, 0xB2, 0xB3, 0xB4,
+        0xC0, 0xC1, 0xC2, 0xC3, 0xC4, 0xC5, 0xC6, 0xC7, 0xC8, 0xC9, 0xCA, 0xCB, 0xCC, 0xCD, 0xCE, 0xCF, 0xD0, 0xD1, 0xD2, 0xD3, 0xD4, 0x00
+    };
+    // clang-format on
+
+    oled_write_P(qmk_logo, false);
+}
+
+static void render_layer(void) {
+    char    layer_str[4];
+    uint8_t layer = get_highest_layer(layer_state);
+    snprintf(layer_str, sizeof(layer_str), "%u", layer);
+
+    oled_write_P(PSTR("Layer: "), false);
+    oled_write(layer_str, false);
+    oled_advance_page(true);
```

Comment:
This all can be condensed down to: 
```suggestion
    oled_write_P(PSTR("Layer: "), false);
    oled_write_ln(get_u8_str(get_highest_layer(layer_state), ' '), false);
```
(this code is also more avr friendly, too)